lib/web_ui/dev/chrome_installer.dart

```diff
@@ -4,19 +4,63 @@
 
 import 'dart:io' as io;
 
+import 'package:args/args.dart';
+import 'package:args/command_runner.dart';
 import 'package:http/http.dart';
 import 'package:meta/meta.dart';
 import 'package:path/path.dart' as path;
 
 import 'environment.dart';
 
-void main(List<String> args) async {
-  Environment.commandLineArguments = args;
-  try {
-    await getOrInstallChrome();
-  } on ChromeInstallerException catch (error) {
-    io.stderr.writeln(error.toString());
+class ChromeInstallerCommand extends Command {
```

Comment:
@yjbanov do you think I should just remove this command?

Reply:
Yep. We can add it later if we need it.

Reply:
Although, if it works already, let's keep it?